### ISSUE TITLE
Latest release schema not compatible with frictionless 5.17.0

### ISSUE THREAD

**Issue by jevansbio** [tdwg/camtrap-dp#381]
If attempting to validate a datapackage using Frictionless 5.17.0 (the latest version of frictionless on pip) using the 1.0 schema, the following error is raised:

`'#$defs/version' is not a valid anchor, neither as a plain name anchor nor as a JSON Pointer. You may have intended to use '#/$defs/version', as the slash is required *before each segment* of a JSON pointer.   '`

Which seems to be related to the naming of the keys in the JSON schema. This error was not present in Frictionless 5.15.0.

**Comment by peterdesmet** [comment]
Thanks! That is something we will have to address in an next version of the standard. @roll is this a new requirement in frictionless-py?

**Comment by roll** [comment]
@peterdesmet 
No, I don't think it was every change but `jsonschema-py` version might have been updated
